output/cloud/expv2/output.go

```diff
@@ -156,14 +150,14 @@ func (o *Output) Start() error {
 		}
 		insightsClient := insights.NewClient(insightsClientConfig)
 
-		ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Why has this increased ten times? :thinking: 

If we are not sure about the value, it seems better to make this configurable

Reply:
I have increased this since 1 second to establish a connection is not that long, considering that our backend is in a single region, but the cloud tests can be distributed geographically. This can be inferred from RTT times across the globe.

Reply:
Ideally, yeah. However, these are already tuned based on a few experiments we did in the past. Let's leave this for the future.